assets/app/scripts/directives/logViewer.js

```diff
@@ -64,6 +64,52 @@ angular.module('openshiftConsole')
               logLinks.scrollTop();
             };
 
+            var bufferedLines = [];
```

Comment:
I was thinking if we moved the flex attributes to CSS, it would actually be a pretty small amount of code. Only three elements... I've been trying to avoid changing the markup so near the end, though.